cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -326,33 +319,40 @@ message GeometryFilter {
     InterpolationMethod interpolation_method = 2; // Only used for linear geometries in 3D
 }
 
-message Seismic2dExtent {
+message Extent {
```

Comment:
I feel like `Extent` might be a bit too vague. What about `TraceExtent`? And then the variants are `TwoDeeTraceExtent` etc.

Not a strong opinion, it could be that `Extent` is fine.

Reply:
Actually, maybe `TraceExtent` is confusing, as it might be taken as filtering a single trace...